substra_materials/titanic_example/quick_start.ipynb

```diff
@@ -14,7 +14,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 25,
+   "execution_count": 1,
```

Comment:
Can you please revert your changes on this file ? So that it is not part of the PR diff